scripts/populateInstances.js

```diff
@@ -1,11 +1,12 @@
 /// TODO: Merge this script with `populateCentralized` once the centralized population is more flexible
 
-const { execSync } = require('child_process')
-
+const util = require('node:util');
+const exec = util.promisify(require('node:child_process').exec);
```

Comment:
What is the value to promisify the exec?

Reply:
It' a promise of the stdout and stderr of the executed command. If there is an error, the promise is rejected